Softeq.XToolkit.Bindings/BindingGeneric.cs

```diff
@@ -1581,6 +1581,30 @@ public void SetConvertBack(Func<TTarget, TSource> convertBack)
             {
                 _convertBack = new WeakFunc<TTarget, TSource>(convertBack);
             }
+
+            private TTo ConvertSafely<TFrom, TTo>(TFrom value)
+            {
+                try
+                {
+                    var notNullableFromType = Nullable.GetUnderlyingType(typeof(TFrom));
+                    object notNullableValue = value;
+                    if (notNullableFromType != null)
+                    {
+                        if (value == null)
+                        {
+                            return default(TTo);
+                        }
+                        notNullableValue = System.Convert.ChangeType(value, notNullableFromType);
+                    }
+
+                    var notNullableToType = Nullable.GetUnderlyingType(typeof(TTo)) ?? typeof(TTo);
```

Comment:
As I can see, this method can be simplified to only 2 last lines, could you please check?

Reply:
you can use string.IsNullOrEmpty()